optaplanner-docs/src/main/assembly/generated-docs-zip.xml

```diff
@@ -1,28 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.1.1"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.1 https://maven.apache.org/xsd/assembly-2.1.1.xsd">
 
   <id>zip-with-generated-docs</id>
   <formats>
-    <format>dir</format>
```

Comment:
Removing the `dir` format breaks https://github.com/kiegroup/optaplanner/blob/main/build/release/upload_distribution.sh#L72 as it copies the documentation HTML and PDF from the directory.

The script should be updated to take them from `target/generated-sources`. To make it even more interesting, the `target/generated-sources/pdf` contains not only the resulting PDF, but also other resources that should not be copied.

Reply:
That's an option, too. On the other hand, why unzip an archive to copy files that are already there?

Reply:
To avoid having to redo all the work I've already done in assembling those things. :-) But I see your point.